tests/e2e/conftest.py

```diff
@@ -170,9 +170,12 @@ def _protostar(args: List[str], ignore_exit_code=False) -> str:
     return _protostar
 
 
-@pytest.fixture(name="devnet_gateway_url", scope="module")
-def devnet_gateway_url_fixture(devnet_port: int):
+@pytest.fixture(name="devnet_gateway_url", scope="function")
```

Comment:
had to change devnet scopes to function. something weird happens when sharing the same devnet instance between migrator tests

Reply:
Maybe we could parallelize those?

Reply:
I heard from the guys that cov has problem with this, so maybe it's not the way to go